e2e_tests/conftest.py

```diff
@@ -35,7 +34,14 @@ def browser_context_args(browser_context_args):
     }
 
 
-@pytest.fixture(autouse=True)
-def apply_block_css_decorator(request, page):
+@pytest.fixture()
+def page(request, page) -> Page:
```

Comment:
The apply_block_css_decorator was doing something weird, essentially making tests run twice which was breaking any test that required loggin in as by that point we would bypass the login page